share/shwap/row_namespace_data_test.go

```diff
@@ -9,68 +9,77 @@ import (
 
 	"github.com/stretchr/testify/require"
 
+	libshare "github.com/celestiaorg/go-square/v2/share"
+
 	"github.com/celestiaorg/celestia-node/share"
 	"github.com/celestiaorg/celestia-node/share/eds"
 	"github.com/celestiaorg/celestia-node/share/eds/edstest"
-	"github.com/celestiaorg/celestia-node/share/sharetest"
 	"github.com/celestiaorg/celestia-node/share/shwap"
 )
 
 func TestNamespacedRowFromShares(t *testing.T) {
 	const odsSize = 8
 
-	minNamespace, err := share.NewBlobNamespaceV0(slices.Concat(bytes.Repeat([]byte{0}, 8), []byte{1, 0}))
-	require.NoError(t, err)
-	err = minNamespace.ValidateForData()
+	minNamespace, err := libshare.NewV0Namespace(slices.Concat(bytes.Repeat([]byte{0}, 8), []byte{1, 0}))
 	require.NoError(t, err)
+	require.NoError(t, minNamespace.ValidateForData())
 
 	for namespacedAmount := 1; namespacedAmount < odsSize; namespacedAmount++ {
-		shares := sharetest.RandSharesWithNamespace(t, minNamespace, namespacedAmount, odsSize)
-		parity, err := share.DefaultRSMT2DCodec().Encode(shares)
+		shares, err := libshare.RandSharesWithNamespace(minNamespace, namespacedAmount, odsSize)
+		require.NoError(t, err)
+		parity, err := share.DefaultRSMT2DCodec().Encode(libshare.ToBytes(shares))
+		require.NoError(t, err)
+
+		paritySh, err := libshare.FromBytes(parity)
 		require.NoError(t, err)
-		extended := slices.Concat(shares, parity)
+		extended := slices.Concat(shares, paritySh)
 
 		nr, err := shwap.RowNamespaceDataFromShares(extended, minNamespace, 0)
 		require.NoError(t, err)
 		require.Equal(t, namespacedAmount, len(nr.Shares))
 	}
 }
 
-func TestNamespacedRowFromSharesNonIncluded(t *testing.T) {
-	// TODO: this will fail until absence proof support is added
-	t.Skip()
-
-	const odsSize = 8
-	// Test absent namespace
-	shares := sharetest.RandShares(t, odsSize)
-	absentNs, err := share.GetNamespace(shares[0]).AddInt(1)
-	require.NoError(t, err)
-
-	parity, err := share.DefaultRSMT2DCodec().Encode(shares)
-	require.NoError(t, err)
-	extended := slices.Concat(shares, parity)
-
-	nr, err := shwap.RowNamespaceDataFromShares(extended, absentNs, 0)
-	require.NoError(t, err)
-	require.Len(t, nr.Shares, 0)
-	require.True(t, nr.Proof.IsOfAbsence())
-}
+// func TestNamespacedRowFromSharesNonIncluded(t *testing.T) {
+// //TODO: this will fail until absence proof support is added
```

Comment:
Should this test be uncommented?

Reply:
nope.  Absence proof should be supported in shwap